packages/sync-actions/test/stores-sync.spec.js

```diff
@@ -45,4 +46,35 @@ describe('Actions', () => {
     const expected = [{ action: 'setLanguages', languages: now.languages }]
     expect(actual).toEqual(expected)
   })
+
+  test('should build `setDistributionsChannels` action', () => {
```

Comment:
it is important to note that `distributionChannels` on a Store is a list of referenced distribution channels.
It is not intended that the update actions are generated to be `changeDistributionChannel`.. instead, we set the list of references